src/components/TransactionFlow/TxReceipt.tsx

```diff
@@ -182,6 +193,20 @@ export default function TxReceipt({
     }
   })();
 
+  const handleTxSpeedUpRedirect = async () => {
+    if (!txConfig) return;
+    const { fast } = await fetchGasPriceEstimates(txConfig.network);
```

Comment:
Error handling?

Reply:
it's handled by that function already :)